change/monosize-bundler-webpack-9e0708c4-c2eb-4634-9f5f-087e2d07975f.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "major",
```

```suggestion
  "type": "minor",
```

these packages are `0.x`